arches/app/templates/views/components/widgets/edtf.htm

```diff
@@ -10,67 +10,84 @@
+          }" tabindex="0" onkeyup="if(event.which == 13 || event.keyCode == 13){ $(this).trigger('click'); }">
+            <i class="fa" data-bind="{
                   css: {
                       'fa-caret-down' : !showEDTFFormats(),
                       'fa-caret-up' : showEDTFFormats(),
                   }
               }"></i>
-              <span data-bind="text: showEDTFFormats() ? '{% trans "EDTF Formats" %}' : '{% trans "EDTF Formats" %}'"></span>
+            <span data-bind="text: showEDTFFormats() ? '{% trans " EDTF Formats" %}' : '{% trans "EDTF Formats" %}'"></span>
           </div>
-          <div class="form-horizontal" data-bind="if: showEDTFFormats">
-              <div class="edtf-style-tools-panel">
-                  <div class="widget-help-panel">
-                    <h2 class="">{% trans "Extended Date/Time Formats (EDTF)" %}</h2>
-                    <h3 class="">
-                      {% trans "The EDTF datatype allows you to describe dates (even uncertain dates).  You can find a summary of the standard here:" %}
-                      <a class="" target="_blank" data-bind="attr: { href: 'https://www.loc.gov/standards/datetime'}, click: function() { return true;}, clickBubble: false">{% trans "EDTF Date Specfication (Library of Congress)" %}</a>
-                    </h3>
-                    <h3 class="">
-                      {% trans "Some common encodings:" %}
-                    </h3>
+          <div class=" form-horizontal" data-bind="if: showEDTFFormats">
+                <div class="edtf-style-tools-panel" tabindex="0">
+                    <div class="widget-help-panel">
+                        <h2>{% trans "Extended Date/Time Formats (EDTF)" %}</h2>
+                        <h3>
+                            {% trans "The EDTF datatype allows you to describe dates (even uncertain dates). You can
+                            find a summary of the standard here:" %}
+                            <a target="_blank"
+                                data-bind="attr: { href: 'https://www.loc.gov/standards/datetime'}, click: function() { return true;}, clickBubble: false">{%
+                                trans "EDTF Date Specfication (Library of Congress)" %}</a>
+                        </h3>
+                        <h3>
+                            {% trans "Some common encodings:" %}
+                        </h3>
 
-                    <dl class="">
-                        <dt class="">2021-04-12 <span class="text-thin">{% trans ' [year][“-”][month][“-”][day]'%}</span></dt>
-                        <dd class="mar-btm">{% trans "Refers to the calendar date 2021 April 12th with day precision" %}</dd>
+                        <dl>
+                            <dt>2021-04-12 <span class="text-thin">{% trans ' [year][“-”][month][“-”][day]'%}</span>
+                            </dt>
+                            <dd class="mar-btm">{% trans "Refers to the calendar date 2021 April 12th with day
```

Comment:
breaking lines midway through trans tags will result in those line breaks being included in the manifests for translated strings, which I think is not desirable.  If we want to add line breaks here, we should not break midway through trans tags.

Reply:
Submitted a commit fixing the new line issues in the trans content